e2e-tests/src/components/CustomTailwindButton.tsx

```diff
@@ -4,6 +4,8 @@ interface Props {
   className?: TailwindClass;
 }
 
+export const initialProps: Props = { className: "" };
```

Comment:
Is this necessary?

Reply:
I added that so the custom tailwind test doesn't need an extra step to set the `className` prop to a default (instead of undefined/disabled) using the undefined menu button. We'd have to do one or the other, and this one seemed slightly simpler? Not a strong preference though

Reply:
Oh gotcha we can keep it!